rca/api_content/templates/api_content/api_content_admin.html

```diff
@@ -0,0 +1,32 @@
+{% extends "wagtailadmin/base.html" %}
+{% load wagtailadmin_tags static i18n %}
+{% block titletag %}{% trans "Dashboard" %}{% endblock %}
+{% block bodyclass %}homepage{% endblock %}
+
+
+{% block content %}
+    <script>
```

Comment:
I do not quite understand why this requires JavaScript rather than just having a normal HTML form. Also, no CSRF check (not that this is crucial in this situation).